app/crypto/key-encryption.ts

```diff
@@ -1,6 +1,6 @@
 import { Buffer } from 'buffer';
 
-const algorithmName = 'AES-CBC';
+const algorithmName = 'AES-CBC' as const;
```

Comment:
What is `as const`? Googling didn't show anything obvious.

Reply:
I thought this was a type widening thing widening thing, but indeed it isn't necessary. Removed.